Models/Mattermost/EventData/MattermostPostedEventDataModel.cs

```diff
@@ -93,6 +93,11 @@ public class MattermostPostedEventDataModel :
 
         public ChatMessageModel ToChatMessageModel(IChatServiceConnection chatServiceConnection)
         {
+            // if we receive a message in a channel with only one other member, we assume it's a private message
```

Comment:
Is there no way to deterministically distinguish private messages?

Reply:
There are, but Mattermost and Slack have different ways of indicating whether it's a private message or not so it wasn't straight forward to me to come up with a way to properly reflect it through the shared data model. This seemed like a valid test according to both API docs that I could reasonably implement for now.

Reply:
Maybe just indicate with `// TODO`

Reply:
added a `// TODO`